addons/l10n_it_edi_sdicoop/models/account_edi_format.py

```diff
@@ -116,7 +116,7 @@ def _l10n_it_post_invoices_step_1(self, invoices):
                 'name': filename,
                 'res_id': invoice.id,
                 'res_model': invoice._name,
-                'datas': base64.encodebytes(xml),
+                'datas': base64.b64encode(xml.encode()),
```

Comment:
we can also use the raw field here.  (where we don't do b64encode)

Reply:
👍🏻